src/components/Modals/DeployEntitiesModal/DeployEntitiesModal.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { Modal } from 'decentraland-dapps/dist/containers'
+import { Button, ModalActions, ModalNavigation } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Props, DeployEntitiesModalView } from './DeployEntitiesModal.types'
+import './DeployEntitiesModal.css'
+
+export default class DeployEntitiesModal extends React.PureComponent<Props> {
```

Comment:
If I'm not wrong, the `ApprovalModal` can be opened in different steps. We might be able to use the `ApprovalFlow` modal directly when clicking the deploy missing entities button, what do you think about it?